### DISCUSSION TITLE
无法使用任务编排

### DISCUSSION THREAD

**Discussion by itlubber** [holdno/gopherCron#33] [Q&A]
我在创建 `任务编排` 时会无法创建，不知道是否是哪里配置有问题，各位大佬可否帮忙看下QaQ

![image](https://github.com/holdno/gopherCron/assets/39397838/48882e0e-bc06-4073-9525-fe45029e0f9d)

**Comment by holdno** [comment]
更新下版本可以解决哈，应该是用了跟前端不匹配的版本导致
